examples/flutter_gallery/lib/demo/contacts_demo.dart

```diff
@@ -136,23 +138,23 @@ class ContactsDemoState extends State<ContactsDemo> {
                     });
                   },
                   itemBuilder: (BuildContext context) => <PopupMenuItem<AppBarBehavior>>[
-                    const PopupMenuItem<AppBarBehavior>(
-                      value: AppBarBehavior.normal,
-                      child: Text('App bar scrolls away')
-                    ),
-                    const PopupMenuItem<AppBarBehavior>(
-                      value: AppBarBehavior.pinned,
-                      child: Text('App bar stays put')
-                    ),
-                    const PopupMenuItem<AppBarBehavior>(
-                      value: AppBarBehavior.floating,
-                      child: Text('App bar floats')
-                    ),
-                    const PopupMenuItem<AppBarBehavior>(
-                      value: AppBarBehavior.snapping,
-                      child: Text('App bar snaps')
-                    ),
-                  ],
+                        const PopupMenuItem<AppBarBehavior>(
```

Comment:
Why does this get indented so far? I thought a trailing comma would always give you two space indent?

Reply:
Great catch. The way dartfmt is handling closures in build methods is close to what Flutter needs but not quite there. I've added a two line fix and indentation for builder methods is now good for flutter code.